UnityProject/Assets/Scripts/Managers/GameManager.cs

```diff
@@ -470,7 +470,7 @@ public void StartRound()
 
 		// Tell all clients that the countdown has finished
 		UpdateCountdownMessage.Send(true, 0);
-		EventManager.Broadcast(Event.PostRoundStarted);
+		EventManager.Broadcast(Event.PostRoundStarted, true);
```

Comment:
This means the event is networked right? 

Can you just go through and check that all the other PostRoundStarted events are checked for IsServer

Reply:
ok will do,  dead test on headless so nothing exploded but I will check

Reply:
yeah is all good, Everything that uses it Check it is server